### ISSUE TITLE
shipit.noChangelog option throws an error

### ISSUE THREAD

**Issue by sethomas** [intuit/auto#1711]
**Describe the bug**

Having `auto.shipit.noChangelog` defined in `package.json` or `.autorc` throws an error

**To Reproduce**

1. make a `package.json` like so:
```
{
  "name": "auto-nochangelog",
  "version": "1.0.0",
  "dependencies": {
    "auto": "^10.6.1"
  },
  "auto": {
    "shipit": {
      "noChangelog": true
    }
  }
}
```

2. run `npx auto shipit --dry-run`
3. notice the error:
```
✖  error     Found configuration errors:

".autorc"

Found unknown configuration keys: shipit.noChangelog
```

or....

1. make a `package.json` like so:
```
{
  "name": "auto-nochangelog",
  "version": "1.0.0",
  "dependencies": {
    "auto": "^10.6.1"
  }
}
```
2. make a `.autorc` like so:
```
{
  "shipit": {
    "noChangelog": true
  }
}
```
3. run `npx auto shipit --dry-run`
4. notice the error:
```
✖  error     Found configuration errors:

".autorc"

Found unknown configuration keys: shipit.noChangelog
```
 
**Expected behavior**

I'd expect `auto.shipit.noChangelog` to work from either `package.json` or `.autorc`.

**Comment by sethomas** [comment]
out of the options listed [here](https://intuit.github.io/auto/docs/generated/shipit#configurable-options) only `onlyGraduateWithReleaseLabel` does not error.

**Comment by adierkens** [comment]
:rocket: Issue was released in [`v10.6.2`](https://github.com/intuit/auto/releases/tag/v10.6.2) :rocket:

**Comment by sethomas** [comment]
Confirmed new version does not error or produce a CHANGELOG.

THANKS
